CHANGELOG.rst

```diff
@@ -2,6 +2,40 @@
 Autopush Changelog
 ==================
 
+1.9.0 (**dev**)
+===============
+
+Features
+--------
+
+Bug Fixes
+---------
+
+WebPush
+-------
+
+1.8.1 (2015-11-16)
+===============
```

Comment:
rst requires the header indicator here to match the line length of the header above it